src/agent.rs

```diff
@@ -1,3 +1,5 @@
+use std::net::SocketAddr;
```

Comment:
nitpik: I think it's better to move this down.

Reply:
Done 🫡